sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1453,6 +1453,13 @@ object SQLConf {
 
   val CONTINUOUS_STREAMING_EPOCH_BACKLOG_QUEUE_SIZE =
     buildConf("spark.sql.streaming.continuous.epochBacklogQueueSize")
+      .doc("The max number of entries to be stored in queue to wait for late epochs. " +
+        "If this parameter is exceeded by the size of the queue, stream will stop with an error.")
+      .intConf
+      .createWithDefault(10)
+
+  val CONTINUOUS_STREAMING_EPOCH_MSG_BACKLOG_QUEUE_SIZE =
+    buildConf("spark.sql.streaming.continuous.epochMessageBacklogQueueSize")
```

Comment:
The doc of `CONTINUOUS_STREAMING_EPOCH_MSG_BACKLOG_QUEUE_SIZE` is a simple copy paste of the doc of `CONTINUOUS_STREAMING_EPOCH_BACKLOG_QUEUE_SIZE`, this does not seems to me right here!

Reply:
get it